cmd/arc/services/api.go

```diff
@@ -3,16 +3,16 @@ package cmd
 import (
 	"context"
 	"fmt"
+	natscore "github.com/bitcoin-sv/arc/internal/message_queue/nats/client/nats_core"
+	natsjetstream "github.com/bitcoin-sv/arc/internal/message_queue/nats/client/nats_jetstream"
+	natsconnection "github.com/bitcoin-sv/arc/internal/message_queue/nats/nats_connection"
```

Comment:
So far, I see we have a standard in Arc codebase of naming packages with an underscore. 

This trick is smart for satisfying the linter, but it's not a long-term solution and it can get annoying when importing new packages.

We should either decide to change the standard and rename all packages without the underscore or remove this linter rule.

I'm in favour of staying with the underscore, it's more readable :)

Reply:
Hello
I agree with @kuba-4chain we should keep the underscores in the package aliases.
@pawellewandowski98 would it be possible to configure revive in a way so that this rule is ignored?

Reply:
Done, I added exclude rule to the config file:

```
issues:
  max-same-issues: 50

  exclude:
    - "don't use an underscore in package name"
```